Lib/Inc/CGpioWrapper.h

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I think this should follow the doxygen style for the file to be included?
```   
/**
/* @file CGpioWrapper.h
/*
```

Reply:
I did not know that. Cool.